src/main/webapp/resources/angular/category/categoryList/category_list_controller.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('crowdfundingApp')
+        .controller('CategoryListController', CategoryListController);
+
+    CategoryListController.$inject = ['CategoryUtils'];
+
+    function CategoryListController(CategoryUtils) {
+
+        var vm = this;
```

Comment:
what does `vm` mean? If you want to store `this` the convention approach it to use variable named `self`